monitoring/pom.xml

```diff
@@ -41,6 +41,9 @@
         <pkg.win.dist>${project.build.directory}/windows</pkg.win.dist>
         <pkg.implementationTitle>ThingsBoard Monitoring Service</pkg.implementationTitle>
         <pkg.mainClass>org.thingsboard.monitoring.ThingsboardMonitoringApplication</pkg.mainClass>
+
+        <californium.version>2.6.1</californium.version>
+        <leshan.version>2.0.0-M4</leshan.version>
```

Comment:
We need specific versions for LwM2M client that differ from the ones in the main pom